app/pandas.ipynb

```diff
@@ -402,9 +523,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# use // operator here"
-   ],
+   "source": "floor_division = s1 // s2",
```

Comment:
The floor division operation is defined but not executed. You should print the result to verify the operation.